src/buildfromsource/FSharp.Compiler.nuget/FSharp.Compiler.nuget.fsproj

```diff
@@ -21,12 +21,12 @@
     <PackageTags       Condition="'$(PackageTags)' == ''"      >Visual F# Compiler FSharp functional programming</PackageTags> 
     <PreReleaseSuffix  Condition="'$(PreRelease)' != 'false'">-rc-$(BuildRevision.Trim())-0</PreReleaseSuffix>
     <PackageVersion>4.2.0$(PreReleaseSuffix)</PackageVersion>
-    <PackageProperties>-prop "licenseUrl=$(PackageLicenceUrl)" -prop "version=$(PackageVersion)" -prop "authors=$(PackageAuthors)" -prop "projectUrl=$(PackageProjectUrl)" -prop "tags=$(PackageTags)"</PackageProperties>
+    <PackageProperties>-prop "licenseUrl=$(PackageLicenceUrl)" -prop "version=$(PackageVersion)" -prop "authors=$(PackageAuthors)" -prop "projectUrl=$(PackageProjectUrl)" -prop "tags=$(PackageTags)" -prop "diasymreaderversion=1.1.0" -prop "diasymreaderportablepdbversion=1.2.0"</PackageProperties>
```

Comment:
Are these the correct versions for build from source?

Reply:
[Yes](https://github.com/Microsoft/visualfsharp/blob/71fcc347dd009d98193619747ee266914977154f/src/buildfromsource/FSharp.Compiler.Private/FSharp.Compiler.Private.fsproj#L639-L640).